packages/cli/src/register-command.ts

```diff
@@ -16,7 +16,7 @@ interface CommonParams {
 export interface ParamDetails {
   prompt?: string;
   promptType?: 'confirm' | 'list' | 'input';
-  choices?: Choice[];
+  choices?: readonly Choice[];
```

Comment:
BRAIN EXPLODES

Reply:
Hehe. I had to add this btw because `const kinds = ['regular', 'upgradeable', 'minimal'] as const;` creates a `readonly` array, because of the `as const`, and if I don't make it so the type will be `string[]`.